buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -86,6 +86,7 @@ SPECS = ['ppc64_aix'      : CURRENT_RELEASES,
          'x86-64_linux_xl': CURRENT_RELEASES,
          'x86-64_linux'   : CURRENT_RELEASES,
          'x86-64_linux_cm': ['8', '11'],
+         'x86-64_linux_jit' : ['8', '11'],
```

Comment:
Is it supported only on 8 and 11?

Reply:
@mpirvu can you confirm what JDK versions JITServer supports?

Reply:
Yes, we only test 8 and 11